src/fragmentarium/ui/fragment/FragmentView.tsx

```diff
@@ -93,6 +94,11 @@ function FragmentView({
       }
       wide
     >
+      <HeadTags
+        title={`${fragment.number}: Fragment edition in the electronic Babylonian Library`}
```

Comment:
Maybe shorten, e.g., `${fragment.number}: eBL Fragment Edition` (though in this case the important bit comes first so it's not a big issue IMO!)